projects/cashmere-examples/src/lib/drawer-double/drawer-double-example.html

```diff
@@ -0,0 +1,21 @@
+<div class="drawer-container">
```

Comment:
Could this be changed to drawer-side-example, since it's main purpose is to show that a drawer can be stationed to the side of the window

Reply:
sure thing, done.